widget/embedded/src/components/ItemPicker/ItemPicker.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface PropTypes {
+  onClick: () => void;
+  value: {
+    label?: string;
+    logo?: string | React.ComponentType<{ size?: number }>;
```

Comment:
You never used `logo` with a react component. I don't know why you've added to the interface.

Reply:
fixed